src/graphql/export.rs

```diff
@@ -2669,8 +2669,9 @@ fn export_statistic_file(
             loop {
                 match iter.next() {
                     Some(Ok(item)) => {
-                        // replace new value (min_index's vector value is always exist)
-                        *iter_next_values.get_mut(min_index).unwrap() = item;
+                        *iter_next_values
+                            .get_mut(min_index)
+                            .expect("min_index's vector value always exists") = item;
```

Comment:
I don't think this message is clear enough. I suggest:

`"The index is one of the actual elements in the vector, so it is always valid."`